cli/assets/templates/zig/build.zig

```diff
@@ -2,41 +2,42 @@ const std = @import("std");
 const expect = std.testing.expect;
 const expectEqualStrings = std.testing.expectEqualStrings;
 
-/// Panics if current zig version is below the minimal required.
-fn check_zig_version() !void {
-    const zig_version = @import("builtin").zig_version;
-    if (zig_version.major > 0) return;
-    if (zig_version.minor > 10) return;
+fn version_supports_stack_first(zig_version: std.SemanticVersion) !bool {
+    if (zig_version.major > 0) return true;
+    if (zig_version.minor > 10) return true;
 
     if (zig_version.minor == 10) {
         if (zig_version.pre) |pre| {
             std.debug.print("{s}", .{pre});
```

Comment:
Oh, my dirty debug code is merged! Remove it if you like.

Reply:
Done